app/src/main/java/woowacourse/movie/feature/bookingdetail/view/BookingDetailActivity.kt

```
@@ -0,0 +1,177 @@
+    private val presenter: BookingDetailPresenter by lazy { BookingDetailPresenter(this) }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setupView()
+        setupDateSpinnerItemClickListener()
+        setupTimeSpinnerItemClickListener()
+        setupTicketCountClickListeners()
+        setupSelectCompleteClickListener()
+        presenter.prepareBookingInfo(
+            intent.getExtra(SCREENING_KEY) ?: throw IllegalArgumentException(),
```

detekt (1.23.7)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)